src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  gap: $spaceBetweenDates;
+  flex-wrap: wrap;
+  margin-top: 30px;
+  width: ($widthsDates * 7) + ($spaceBetweenDates * 6);
+  padding: $paddings;
+
+  &__date {
+    height: 100px;
+    box-sizing: border-box;
```

Comment:
It's the general rule that will be applied for all elements inside your project, so will be better to set it using `*` selector